### ISSUE TITLE
ML namelist additions

### ISSUE THREAD

**Issue by Katetc** [ESCOMP/PUMAS#62]
The review for bringing in the PUMAS ML code asked if the **ncd** parameter in stochastic_tau_cam.F90 and stochastic_emulated_cam.F90 should be exposed as a namelist field in CAM. Because this is used a dimension for output history arrays and possibly other arrays, I was hesitant to do so. However, I would be interested to hear if others thought this was a good or bad idea. @djgagne @andrewgettelman @wkchuang 

I may use this thread for other questions related to the review and PR of bringing this into CAM. Thanks!

**Comment by andrewgettelman** [comment]
No. Not user modifiable unless you know what you are doing. Not namelist.
ncd = number of bins I think for the Tau code. I believe this is fixed by the prescribed collection kernel.
